api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -206,6 +214,22 @@ public async Task<IActionResult> AddAsync(ContentModel model)
         var newContent = (Content)model;
         newContent.OwnerId = user.Id;
         newContent.PostedOn = newContent.Status == ContentStatus.Publish || newContent.Status == ContentStatus.Published ? DateTime.UtcNow : null;
+
+        var topicScore = TopicScoreHelper.GetScore(_topicScoreRuleService.FindAll(), model.PublishedOn?.TimeOfDay, model.SourceId, model.Body.Length, model.Section, model.Page, model.SeriesId);
+        if (topicScore != null) {
```

Comment:
Is there a reason this extra logic isn't the same in all places?  Also could it be moved to the `TopicScoreHelper` to centralize it?